packages/one-app-bundler/utils/extendWebpackConfig.js

```diff
@@ -29,6 +29,16 @@ function getCustomWebpackConfigPath(options, bundleTarget) {
   return false;
 }
 
+function parseProvidedExternals(providedExternals) {
+  return Array.isArray(providedExternals)
```

Comment:
Maybe we can move this function to a utils file, as it seems to be used quite a bit?

Reply:
it's only being used in this file at the moment